src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,29 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    private List<Truck> trucks = new ArrayList<>();
+
+    @Override
+    public List<Truck> get() {
+        Truck truck1 = new Truck();
+        Truck truck2 = new Truck();
+        Truck truck3 = new Truck();
+        trucks.add(truck1);
+        trucks.add(truck2);
+        trucks.add(truck3);
+        truck1.setColor("RED");
+        truck1.setName("Car transporters");
+        truck1.setCurbWeight(5000);
+        truck2.setColor("BLUE");
+        truck2.setName("Cement truck");
+        truck2.setCurbWeight(8000);
+        truck3.setColor("YELLOW");
+        truck3.setName("Box truck");
+        truck3.setCurbWeight(2500);
+        return trucks;
```

Comment:
The size of a list passed to `get()` method should remain the same. You should not add elements to the 'trucks' field each time `get()` is called. This will cause the list to grow indefinitely with each call.